### PR TITLE
SCR:COREWEB-789 - Changes to address spellcheck in non browser UI

### PR DESCRIPTION
**Pull request by SuditiL**
Spellcheck on multiline email body in InteractionDesktop was broken because a line break was getting removed.

Note: Exports are removed from Squire-raw.js because source/exports.js is not there in RAKEFILE

### CODE REVIEW

File: build/squire-raw.js [hunk 1]
```diff
@@ -5190,47 +5192,6 @@ proto.makeUnorderedList = command( 'modifyBlocks', makeUnorderedList );
 proto.makeOrderedList = command( 'modifyBlocks', makeOrderedList );
 proto.removeList = command( 'modifyBlocks', removeList );
 
-// Node.js exports
```
Comment (stormojm) [inline build/squire-raw.js]: Maybe my last email was not clear, but we still need these.  The rakefile has to be updated to match the makefile

File: source/Range.js [hunk 2]
```diff
@@ -213,6 +213,7 @@ var insertTreeFragmentIntoRange = function ( range, frag, root ) {
         // Remove trailing <br> – we don't want this considered content to be
         // inserted again later
         cleanupBRs( block, root, false );
+        fixCursor(block, root);
```
Comment (stormojm) [inline source/Range.js:217]: consider adding a comment why this was necessary

File: build/squire-raw.js (hunk 1 above)
Comment (SuditiL) [reply build/squire-raw.js]: changed with e98ec3d

File: source/Range.js (hunk 2 above)
Comment (SuditiL) [reply source/Range.js:217]: changed with 4359e11

**Pull request merged**